supportedBackends/google/pipelines/v2alpha1/src/test/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionCommandsSpec.scala

```diff
@@ -18,11 +18,42 @@ class ActionCommandsSpec extends FlatSpec with Matchers with Mockito {
       s"flag is $flag"
     }
     
-    recovered shouldBe "flag is  2> gsutil_output.txt; RC_GSUTIL=$?; if [ \"$RC_GSUTIL\" = \"1\" ]; then\n grep \"Bucket is requester pays bucket but no user project provided.\" gsutil_output.txt && echo \"Retrying with user project\"; flag is -u my-project; fi "
+    recovered shouldBe """flag is  2> gsutil_output.txt
+                         |# Record the exit code of the gsutil command without project flag
+                         |RC_GSUTIL=$?
+                         |if [ "$RC_GSUTIL" != "0" ]; then
+                         |  echo "gsutil command failed"
+                         |  # Print the reason of the failure to stderr
+                         |  cat gsutil_output.txt 1>&2
```

Comment:
This is the real value of this PR, the rest is just switching to multiline bash for clarity